src/khoj/interface/web/chat.html

```diff
@@ -2170,6 +2214,24 @@
                             </svg>
                         </button>
                         <textarea id="chat-input" class="option" oninput="onChatInput()" onkeydown=incrementalChat(event) autofocus="autofocus" placeholder="Type / to see a list of commands"></textarea>
+                        <!-- Shortcut Handler for Accessing Old Messages -->
+                        <script>
+                            document.addEventListener('keydown', function(event) {
+                                const chatInput = document.getElementById('chat-input');
+                                if (document.activeElement === chatInput) {
```

Comment:
Can this be simplified to something like below:
```suggestion
                            let chatInput = document.getElementById('chat-input');
                            chatInput.addEventListener('keydown', function(event) {
```